src/core/AutomationClip.cpp

```diff
@@ -1041,13 +1041,13 @@ std::vector<AutomationClip *> AutomationClip::clipsForModel(const AutomatableMod
 
 
 AutomationClip * AutomationClip::globalAutomationClip(
-							AutomatableModel * _m )
+							AutomatableModel * _m)
```

Comment:
Join this with previous line (and move the `*` to the typename in the return type), and let's rename `_m` to something like `model`
```suggestion
							AutomatableModel* _m)
```